argocd/app-of-apps/values.tmpl.yaml

```diff
@@ -62,7 +62,7 @@ argo-cd:
       enabled: true
   server:
     config:
-      url: https://argocd.apps.${cluster_name}.${base_domain}
+      url: ${argocd.domain}
```

Comment:
I guess the scheme is mandatory.

Reply:
Interesting. It actually works this way in katacoda it seems

Reply:
Fixed